.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pyc
 .idea
+*.sqlite3
```

Comment:
:+1: for removing the database from the repo!

specifically `db.sqlite3`, right? Should we update this to `db.sqlite3`?

For whatever reason, if we ever wanted to add another .sqlite3 file, git would ignore it... is that what we want?